static/app/views/dashboards/widgetBuilder/components/newWidgetBuilder.spec.tsx

```diff
@@ -72,7 +72,14 @@ describe('NewWidgetBuilder', () => {
 
     MockApiClient.addMockResponse({
       url: '/organizations/org-slug/events-stats/',
-      body: [],
+      body: {
```

Comment:
needed to add a mock return because this updates some tests to use the new visualization, which throws an error on empty plottables.